constrained-optimization-and-backtesting.qmd

```diff
@@ -27,10 +27,8 @@ tidy_finance <- dbConnect(
 )
 
 industry_returns <- tbl(tidy_finance, "industries_ff_monthly") |>
+  select(-month) |>
```

Comment:
select(NoDur, Durbl, Manuf, Enrgy, HiTec, Telcm, Shops, Hlth, Utils, Other) seems excessive, especially because the script never explicitly calls any of the columns by name. 

Commit [8ca5016](https://github.com/tidy-finance/website/pull/42/commits/8ca5016768c18ea3ac62a068cdadb2764407c660) proposes to print the object instead.

Reply:
Hm fine. There is a reason why SQL does not allow this dplyr selection procedure. It is just bad practice from a DB management perspective to be obscure about columns.